docs/src/backend/07_connecting_the_database.md

```diff
@@ -124,7 +124,7 @@ async fn version(db: actix_web::web::Data<sqlx::PgPool>) -> String {
 
 There are a couple of things going on here, so let's break it down.
 
-First of all, it's worth noticing that **we changed the return type** of the function to `String`. This obbeys to two different reasons:
+First of all, it's worth noticing that **we changed the return type** of the function to `String`. This is for two different reasons:
```

Comment:
This was the only change I couldn't quite figure out what "obbeys" was trying to say, hope the semantics make sense.

Reply:
Yes, totally makes sense, Thank you for the fix @robjtede :heart: